plugins/dell/fu-plugin-dell.h

```diff
@@ -34,8 +37,8 @@ struct tpm_status {
 	guint32 status;
 	guint32 flashes_left;
 };
-#define TPM_EN_MASK	0x0001
-#define TPM_OWN_MASK	0x0004
-#define TPM_TYPE_MASK	0x0F00
-#define TPM_1_2_MODE	0x0001
-#define TPM_2_0_MODE	0x0002
+#define TPM_EN_MASK 0x0001
+#define TPM_OWN_MASK 0x0004
+#define TPM_TYPE_MASK 0x0F00
+#define TPM_1_2_MODE 0x0001
+#define TPM_2_0_MODE 0x0002
```

Comment:
This requires `clang-format off` to protect the current formatting, isn't it?

Reply:
I just check that this is now supported on clang-format-9.0 with AlignConsecutiveMacros: true

> > So here's the problem. Clang format does have a way to correct the structs but it will have two limitations:
> 
> > it also impacts other code in functions. I think this is actually OK - it does a pretty good job.
> > Pointer alignment gets out of wack. It's fixed with https://reviews.llvm.org/D27651 but that's not yet merged into clang-format upstream even.
> > You can see what happens with that turned on in above new sample.
> 
> > It seems this has been fixed with https://reviews.llvm.org/D103245
> 
> Can you check what version of clang format that's in? We may want to only run the pre-commit and hook with a new enough version.

This should be on clang-format-13 onwards